client/executor/runtime-test/src/lib.rs

```diff
@@ -1,5 +1,4 @@
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(feature = "strict", deny(warnings))]
```

Comment:
Why did you remove this?

Reply:
[See this comment](https://github.com/paritytech/substrate/pull/6267#discussion_r436165804)